### PR TITLE
Proposal: WACI-Presentation-Exchange Work Item

### PR DESCRIPTION
**Pull request by brentzundel**
@OR13
Signed-off-by: Brent Zundel <brent.zundel@gmail.com>

### CODE REVIEW

**Comment by rado0x54** [comment]
I think we probably only merge this on or after the first Workitem Call. I'll add some comments to the PR.

File: work_items/WACI_presentation_exchange.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+a variety of credentials formats. 
+
+## Deliverables
+- **Does your work item have any deliverables? These include code projects,
+  standards-track specifications, reports, registries, summaries, blog posts,
+  and anything else that is instantiable and publishable in written form to a
+  broader audience.**
+
+We are writing a standards-track specification.
+
```
Comment (OR13) [inline work_items/WACI_presentation_exchange.md:22]: We will provide example payloads (test vectors) associated with the spec.

File: work_items/WACI_presentation_exchange.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+- **What's the expected timeline?**
+
+We will release v0.1 for developer use by .
```
Comment (OR13) [inline work_items/WACI_presentation_exchange.md:25]:
```suggestion
We will release v0.1 for developer use by June 1 2021
```

File: work_items/WACI_presentation_exchange.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+We will have a standard protocol for requesting and submitting a presentation of
+a variety of credentials formats. 
+
+## Deliverables
+- **Does your work item have any deliverables? These include code projects,
+  standards-track specifications, reports, registries, summaries, blog posts,
+  and anything else that is instantiable and publishable in written form to a
+  broader audience.**
+
+We are writing a standards-track specification.
```
Comment (rado0x54) [inline work_items/WACI_presentation_exchange.md:21]: Should we specifically mention test-vectors or interop profiles here or are these covered via https://github.com/decentralized-identity/org/blob/master/Org%20documents/Membership%20agreements/Project%20Charter%20Addendum.md

**Review by OR13 — CHANGES_REQUESTED** [review]
@brentzundel please commit us to the proposed spec complete date of June 1.

File: work_items/WACI_presentation_exchange.md (hunk 2 above)
Comment (rado0x54) [inline work_items/WACI_presentation_exchange.md:25]: Can we move in the preliminary timeline from IIW?

File: work_items/WACI_presentation_exchange.md (hunk 3 above)
Comment (OR13) [inline work_items/WACI_presentation_exchange.md:21]:
```suggestion
We are writing a standards-track specification, which will include a complete set of test vectors covering all messages exchanged by the protocol.
```

Comment (OR13) [reply work_items/WACI_presentation_exchange.md:21]: Proposed: https://github.com/decentralized-identity/claims-credentials/pull/5/files#r620548576

File: work_items/WACI_presentation_exchange.md (hunk 2 above)
Comment (OR13) [reply work_items/WACI_presentation_exchange.md:25]: Pasting from the notes:

```
May 1: GHP Drafting Groups First Drafts Due - 

            June 1 GHP Interoperability Blueprint Recommendations Spec complete
30 day vision
90 day vision  <- would be ideal to have something that can be tested against for this timeframe. 
180 day vision

            July 1 Test Complete

            August 1 - 10+ Implementations / Vendors passing VP Exchange Interop Tests.
	
 	October 1 - Cross Wallet Interop Exchange Tests.
```

**Review by OR13 — APPROVED** [review]

**Comment by csuwildcat** [comment]
Are we really going to call this thing "XYZ Presentation Exchange" when we have a spec called Presentation Exchange? I feel like this could really confuse people....

**Comment by OR13** [comment]
@csuwildcat current working title is WACI Presentation Exchange... because its about wallets..... 

https://specs.bloom.co/wallet-and-credential-interactions/

WACI is short for "Wallet And Credential Interactions"

File: work_items/WACI_presentation_exchange.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+  standards-track specifications, reports, registries, summaries, blog posts,
+  and anything else that is instantiable and publishable in written form to a
+  broader audience.**
+
+We are writing a standards-track specification, which will include a complete
+set of test vectors covering all messages exchanged by the protocol.
+
+- **What's the expected timeline?**
+
+We will release v0.1 for developer use by June 1 2021
```
Comment (brentzundel) [inline work_items/WACI_presentation_exchange.md:26]:
```suggestion
We will release v0.1 for developer use by June 1 2021, then continue working toward v1.0 with timeline TBD.
```

File: work_items/WACI_presentation_exchange.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+- **Who's the audience of your deliverables?**
+
+We are writing for developers who need to ask for and exchange claims that are
+verifiable, such as Verifiable Credentials.
```
Comment (brentzundel) [inline work_items/WACI_presentation_exchange.md:31]:
```suggestion
verifiable, such as Verifiable Credentials expressed in a Verifiable Presentation.
```

File: work_items/WACI_presentation_exchange.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+- **What are related resources or standards from DIF or adjacent
+  organizations?**
+
+JSON-LD Signatures, JWT, DIF Presentation-Exchange, DIDComm, and WACI.
```
Comment (brentzundel) [inline work_items/WACI_presentation_exchange.md:36]:
```suggestion
JSON-LD Signatures, JWT, DIF Presentation-Exchange and Credential Manifest, DIDComm,
CCG VC HTTP API, GNAP, DPoP, OIDF, SIOP, and WACI.
```

File: work_items/WACI_presentation_exchange.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+- **Where are the notes and recordings published, if any?**
+
+The recordings are published in the #general channel on the DIF Slack. There
+are no notes, however automated transcription is a possibility and actions are
+mainly taken in GitHub issues.
+
+- **Who might be interested in attending?**
+
+Anyone interested in a protocol for asking for and receiving presentations.
```
Comment (brentzundel) [inline work_items/WACI_presentation_exchange.md:65]:
```suggestion
Anyone interested in a protocol for using credentials.
```

File: work_items/WACI_presentation_exchange.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
+# WACI Presentation Exchange
+
+
+## Work Item Owners
+- Orie Steele (@OR13)
+- Brent Zundel (@brentzundel)
```
Comment (brentzundel) [inline work_items/WACI_presentation_exchange.md:6]:
```suggestion
- Brent Zundel (@brentzundel)
- Jace Hensley (@JaceHensley)

```

**Comment by brentzundel** [comment]
@csuwildcat @rado0x54 @wyc any objections to merging this?

**Review by wyc — APPROVED** [review]

**Pull request merged**